### ISSUE TITLE
[Sample Cards]: Talkback does not announce the position of the menu items present in the dropdown appearing after activating Sample Cards button.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7391]
https://microsoft.visualstudio.com/OS/_workitems/edit/38003332
